doc/modules/cassandra/pages/faq/index.adoc

```diff
@@ -79,7 +79,10 @@ intensive process that may result in adverse cluster performance. It's
 highly recommended to do rolling repairs, as an attempt to repair the
 entire cluster at once will most likely swamp it. Note that you will
 need to run a full repair (`-full`) to make sure that already repaired
-sstables are not skipped.
+sstables are not skipped. You should use ConsistencyLevel.QUORUM or ALL
```

Comment:
may be ... `ConsistencyLevel.QUORUM` or `ALL` ...

Reply:
It's fixed